vignettes/test/rendering.Rmd

```diff
@@ -185,3 +185,54 @@ Blablablablablabla.
 # This section is unnumbered {-}
 
 There should however be no bug here!
+
+## Results in tabset {.tabset .tabset-pills}
+
+### Tab 1
+
+blablablabla
+
+```r
+1 + 1
+```
+
+### Tab 2
+
+blop
+
+
+## Results in tabset, no pills {.tabset}
+
+### A thing
+
+something nice
+
+```{r}
+plot(1:42)
+```
+
+#### cool
+
+yay
+
+### Tab 2 {.active}
+
+hello
+
+## Results in tabset, fading {.tabset .tabset-fade}
```

Comment:
This looks no different to me. Is it really something we need to support?

Reply:
They have a fading behaviour when switching from one to the other. I added that because R Markdown supports it and it is in Bootstrap.

Reply:
Yeah I'll wait for at least another bug report. :wink: